adapters/rubicon/rubicon.go

```diff
@@ -601,6 +602,22 @@ func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *
 	return bids, nil
 }
 
+func resolveVideoSizeId(placement openrtb.VideoPlacementType, instl int8) (sizeID int) {
+	if placement != 0 {
+		if placement == 1 {
+			return 201
```

Comment:
Please add JSON test cases to cover this part of the code as well where the `SizeID` is determined based on `imp.Video.Placement`